conf/base.config

```diff
@@ -10,58 +10,64 @@
+    resourceLimits = [
+        cpus: params.max_cpus,
+        memory: params.max_memory,
+        time: params.max_time,
+    ]
 
-    errorStrategy = { task.exitStatus in ((130..145) + 104) ? 'retry' : 'finish' }
-    maxRetries    = 3
-    maxErrors     = '-1'
+    cpus           = { 1 * task.attempt }
+    memory         = { 6.GB * task.attempt }
+    time           = { 4.h * task.attempt }
 
-    withLabel:process_single {
-        cpus   = { check_max( 1                  , 'cpus'    ) }
-        memory = { check_max( 6.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 4.h  * task.attempt, 'time'    ) }
+    errorStrategy  = { task.exitStatus in ((130..145) + 104) ? 'retry' : 'finish' }
+    maxRetries     = 3
+    maxErrors      = '-1'
+
+    withLabel: process_single {
+        cpus   = { 1 }
+        memory = { 6.GB * task.attempt }
+        time   = { 4.h * task.attempt }
     }
-    withLabel:process_low {
-        cpus   = { check_max( 2     * task.attempt, 'cpus'    ) }
-        memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 4.h   * task.attempt, 'time'    ) }
+    withLabel: process_low {
+        cpus   = { 2 * task.attempt }
+        memory = { 12.GB * task.attempt }
+        time   = { 4.h * task.attempt }
     }
-    withLabel:process_medium {
-        cpus   = { check_max( 6     * task.attempt, 'cpus'    ) }
-        memory = { check_max( 36.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 8.h   * task.attempt, 'time'    ) }
+    withLabel: process_medium {
+        cpus   = { 6 * task.attempt }
+        memory = { 36.GB * task.attempt }
+        time   = { 8.h * task.attempt }
     }
-    withLabel:process_high {
-        cpus   = { check_max( 12    * task.attempt, 'cpus'    ) }
-        memory = { check_max( 72.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 16.h  * task.attempt, 'time'    ) }
+    withLabel: process_high {
+        cpus   = { 12 * task.attempt }
+        memory = { 72.GB * task.attempt }
+        time   = { 16.h * task.attempt }
     }
-    withLabel:process_long {
-        time   = { check_max( 20.h  * task.attempt, 'time'    ) }
+    withLabel: process_long {
+        time = { 20.h * task.attempt }
     }
-    withLabel:process_high_memory {
-        memory = { check_max( 200.GB * task.attempt, 'memory' ) }
+    withLabel: process_high_memory {
+        memory = { 200.GB * task.attempt }
     }
-    withLabel:error_ignore {
+    withLabel: error_ignore {
         errorStrategy = 'ignore'
     }
-    withLabel:error_retry {
+    withLabel: error_retry {
         errorStrategy = 'retry'
         maxRetries    = 2
     }
-    withName:CUSTOM_DUMPSOFTWAREVERSIONS {
+    withName: CUSTOM_DUMPSOFTWAREVERSIONS {
         cache = false
     }
 
-    withName:ALIGN_BWAMEM2 {
-        cpus   = { check_max( 12    * task.attempt,                  'cpus'  ) }
+    withName: ALIGN_BWAMEM2 {
```

Comment:
Aren't module specific settings supposed to be in `module.config`? 🤔

Reply:
Yes... I missed this one!